### PR TITLE
Add ruleset for SCS Software homepage and blog

### PR DESCRIPTION
**Pull request by netravnen**

### CODE REVIEW

File: src/chrome/content/rules/Scssoft.com.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+		- blog
+		- www
+
+-->
+<ruleset name="scssoft.com">
+
+	<target host="scssoft.com" />
+	<target host="www.scssoft.com" />
+	<target host="blog.scssoft.com" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Scssoft.com.xml:14]:
```suggestion
	<target host="scssoft.com" />
	<target host="www.scssoft.com" />
	<target host="blog.scssoft.com" />
	<target host="chat.scssoft.com" />
	<target host="eshop.scssoft.com" />
	<target host="forum.scssoft.com" />
	<target host="ks.scssoft.com" />
	<target host="mail.scssoft.com" />
	<target host="modding.scssoft.com" />
	<target host="qa.scssoft.com" />
	<target host="sendy.scssoft.com" />
```

File: src/chrome/content/rules/Scssoft.com.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!--
+	SCS Software
+
+	Fully covered subdomains:
+
+		- blog
+		- www
+
+-->
```
Comment (pipboy96) [inline src/chrome/content/rules/Scssoft.com.xml:9]: This comment is redundant.

```suggestion
```

**Review by pipboy96 — CHANGES_REQUESTED** [review]

File: src/chrome/content/rules/Scssoft.com.xml (hunk 1 above)
Comment (netravnen) [reply src/chrome/content/rules/Scssoft.com.xml:5]: 🆗

File: src/chrome/content/rules/Scssoft.com.xml (hunk 2 above)
Comment (netravnen) [reply src/chrome/content/rules/Scssoft.com.xml]: 🆗

File: src/chrome/content/rules/Scssoft.com.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<ruleset name="scssoft.com">
```
Comment (J0WI) [inline src/chrome/content/rules/Scssoft.com.xml:1]: Cloud you also add [comments for non-working subdomains](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts)?
```
huntingunlimited.scssoft.com
karolina.scssoft.com
www2.scssoft.com
```

**Pull request closed without merging**